Library/Formula/walkmod.rb

```diff
@@ -0,0 +1,22 @@
+class Walkmod < Formula
+  desc "Java-based project to apply and share code conventions"
+  homepage "http://www.walkmod.com"
+  url "https://bitbucket.org/rpau/walkmod/downloads/walkmod-2.0-installer.zip"
+  sha256 "06f11d9c4acfe3e3e0d20316a49ad613cbd526368308c68670ce749b472fc35f"
+  depends_on :java
```

Comment:
Sorry, but if I put an empty line between the `sha256` and the `depends_on` the execution of `brew audit --strict --online walkmod.rb` gives me the following error:
== walkmod.rb ==
C:  6:  1: Trailing whitespace detected.

The java version for walkmod is 1.6, but some plugins requires 1.7. So, to avoid user mistakes, I have setted 1.7.